README.md

```diff
@@ -1,2 +1,21 @@
 # hw01
 Repository for homework 1
+# Short Bio on Tarryn Rennie 
+
+### I decided that for this assignment I was going to tell you **15 things** that you wouldn't know just from meeting me, instead of just the usual tired-out blurb. 
```

Comment:
Looks like you tried to do bold text around 15 things. Your syntax looks correct, but I think it didn't work because your three hashtags at the beginning of the line already made this sentence a bolded level 3 header.